doc/overviews/rate-limiting.md

```diff
@@ -44,54 +44,6 @@ The limit definitions (`limits`) can be declared at the top-level level of the s
   </tbody>
 </table>
```

Comment:
Are we missing a link to the reference page, similar to the other overview pages?